internal/commands/user/disable.go

```diff
@@ -47,9 +47,9 @@ func (cmd *CommandDisable) Handler(profile *cli.Profile, ui terminal.UI) error {
 		return appErr
 	}
 
-	users, usersErr := cmd.realmClient.FindUsers(app.GroupID, app.ID, realm.UserFilter{IDs: cmd.inputs.Users})
-	if usersErr != nil {
-		return usersErr
+	users, userErr := cmd.inputs.resolveUsers(ui, cmd.realmClient, app)
```

Comment:
same thing here with the unnecessary diff... probably copy/pasta across these modified commands if you want to revert those back